client/src/app/videos/+video-edit/shared/video-edit.component.html

```diff
@@ -114,6 +114,11 @@
               </div>
             </div>
 
+            <div class="form-group">
+              <label i18n for="producedAt">Production date</label>
+              <input type="text" id="producedAt" formControlName="producedAt" />
```

Comment:
Here we may put the same type as the `schedulePublicationAt` field.

Reply:
instead of `schedulePublicationAt` may I suggest `schedulePublishedAt`

Reply:
or perhaps instead of `schedulePublishedAt` just `publishedAt`, since the other field is `producedAt`

Reply:
Unless this will be used to schedule when the video will be viewable by visitors, in that case it's fine I think, maybe in that case we'd have both `schedulePublicationAt` and `publishedAt`, one for when it is viewable and the other as a datetime stamp of when it was published at the original place it was uploaded, like youtube for instance.

Reply:
> instead of `schedulePublicationAt` may I suggest `schedulePublishedAt`

`schedulePublicationAt` is already in production if I'm not wrong.

Reply:
> or perhaps instead of `schedulePublishedAt` just `publishedAt`, since the other field is `producedAt`

`publishedAt` is already used for when you make the video public : https://github.com/Chocobozzz/PeerTube/commit/2922e048de95738b3319054ce0778f873a34a0ee